app/models/user.rb

```diff
@@ -10,6 +10,14 @@ class User < ApplicationRecord
   has_many :completed_lessons, through: :lesson_completions, source: :lesson
   has_many :projects
 
+  def format_completion_date
```

Comment:
again because of the way that the best_in_place gem works, it needs to be in user.rb